### PR TITLE
Suppress save before starting notebook debug session.

### LINKED ISSUE #6898: Run line by line in an untitled file prompts to save file
Testing microsoft/vscode#129467

https://user-images.githubusercontent.com/35271042/127231501-0593e8d0-c401-4e40-8771-df7877e39333.mp4

I'm not sure what is expected here, but I got prompted to save an untitled notebook (with no workspace) but also clicked "cancel" and then it kept debugging? Either it should only work if you save it or allow to debug (I vote for the latter).

### PR DESCRIPTION
**Pull request by roblourens**
So we can debug untitled notebooks cleanly, and it's not needed for other unsaved notebooks anyway.
Fix #6898

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [x] ~~Has a [news entry]~~(https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [x] Appropriate comments and documentation strings in the code.
-   [x] ~~Has sufficient logging.~~
-   [x] ~~Has telemetry for enhancements.~~
-   [x] ~~Unit tests & system/integration tests are added/updated.~~
-   [x] ~~[Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.~~
-   [x] ~~[`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).~~

### CODE REVIEW

**Comment by roblourens** [comment]
I just pushed this option to vscode so it isn't in an Insiders build yet

**Review by DavidKutu — APPROVED** [review]

File: src/client/debugger/jupyter/debuggingManager.ts [hunk 1]
```diff
@@ -304,7 +304,10 @@ export class DebuggingManager implements IExtensionSingleActivationService, IDeb
             __mode: mode,
             __cellIndex: cell.index
         };
-        const opts = mode === KernelDebugMode.RunByLine ? { debugUI: { simple: true } } : undefined;
+        const opts: DebugSessionOptions | undefined =
+            mode === KernelDebugMode.RunByLine
+                ? { debugUI: { simple: true }, suppressSaveBeforeStart: true }
```
Comment (DonJayamanne) [inline src/client/debugger/jupyter/debuggingManager.ts:309]: @roblourens
Is this avaialble in VS Code 1.60 stable?

Comment (roblourens) [reply src/client/debugger/jupyter/debuggingManager.ts:309]: No

Comment (roblourens) [reply src/client/debugger/jupyter/debuggingManager.ts:309]: But it will be ignored and not cause an issue, if that's what you're wondering

**Pull request merged**